.github/workflows/pr.yml

```diff
@@ -9,22 +9,29 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: postgres
-        env:
-          POSTGRES_PASSWORD: postgres
+        - image: node:14.x
+          env:
+            PG_HOST: localhost
+            PG_USER: ubuntu
```

Comment:
there is a lot of stuff in this PR that is happening that isn't explained in the PR description. Can we remove everything that doesn't fix the issue?